CONTRIBUTING.md

```diff
@@ -177,6 +177,32 @@ To maintain understandable code and avoid circular dependencies it is important
 - **templates:** Golang templates for generating the html output.
 - **vendor:** External code that Gitea depends on.
 
+## API v1
+
+The API is documented by [swagger](http://try.gitea.io/api/swagger) and is based on [GitHub API v3](https://developer.github.com/v3/).
+Thus, Gitea´s API should use the same endpoints and fields as GitHub´s API as much as possible, unless there are good reasons to deviate. If GitHub doesn't provide a functionality Gitea does, an new endpoint can cover this. If information is on Gitea but not on GitHub API a new field is used which doesn't collide with GitHub one.
```

Comment:
Final nit, breaking up into different lines to maybe make the different suggestions more clear.
```suggestion
Thus, Gitea´s API should use the same endpoints and fields as GitHub´s API as far as possible, unless there are good reasons to deviate.  
If Gitea provides functionality that GitHub does not, a new endpoint can be created.  
If information is provided by Gitea that is not provided by the GitHub API, a new field can be used that doesn't collide with any GitHub fields.
```